test/Dialect/XTenNN/loadExternalConst.mlir

```diff
@@ -0,0 +1,22 @@
+
+// RUN: aten-opt %s -split-input-file -verify-diagnostics
+
+func.func @valid_LEC_op() -> tensor<1x2xf32> {
+    %result = xten_nn.load_external_const { key = "myLayerName", file = "quant_params.h5"} -> tensor<1x2xf32>
```

Comment:
Liam's comment: we want to keep it general and not suggest a file format, so maybe we can give a generic name.
```suggestion
    %result = xten_nn.load_external_const { key = "myLayerName", file = "weights.data"} -> tensor<1x2xf32>
```